tests/rt.sh

```diff
@@ -973,12 +984,21 @@ if [[ -e fail_test ]]; then
     echo "${failed_test_name}"
     echo "${failed_test_name}"    >> ${REGRESSIONTEST_LOG}
   done < fail_test
-   echo ; echo REGRESSION TEST FAILED
-  (echo ; echo REGRESSION TEST FAILED)         >> ${REGRESSIONTEST_LOG}
+  if [[ $COMPILE_ONLY == true ]]; then
+     echo ; echo COMPILE-ONLY REGRESSION TEST FAILED
+    (echo ; echo COMPILE-ONLY REGRESSION TEST FAILED)         >> ${REGRESSIONTEST_LOG}
```

Comment:
Is it needed to be this specific? I would think just "REGRESSION TEST FAILED" is sufficient ?

Reply:
I thought that it would be good to indicate to the user that this was not a full regression test run, but only the compile part of it. No strong preference, though.

Reply:
I'm re-working logs here: #2061 . I could add a section that adds the options used to the log. Also could use an outside opinion on it if you have time. I'm OK keeping this for now while discussions continue on 2061.